browser/src/Input/Mouse.ts

```diff
@@ -61,7 +61,10 @@ export class Mouse extends EventEmitter {
                 } else {
                     scrollcmdY += `ScrollWheelDown>`
                 }
-                this.emit("mouse", scrollcmdY)
+                this.emit("mouse",scrollcmdY + `<${line},${column}>`)
```

Comment:
Also if you run `npm run test` locally, do you get any tslint errors? I wonder if there is something wrong with the validation on travisCI.